exporter/diagnostic_data_collector_test.go

```diff
@@ -68,8 +67,8 @@ mongodb_oplog_stats_wt_transaction_update_conflicts 0` + "\n")
 		"mongodb_oplog_stats_wt_btree_fixed_record_size",
 		"mongodb_oplog_stats_wt_transaction_update_conflicts",
 	}
-	// TODO: use NewPedanticRegistry when mongodb_exporter code fulfils its requirements (https://jira.percona.com/browse/PMM-6630).
```

Comment:
Why did you replace it by PedanticRegistry? Did you make it stable?

Reply:
Metric names are stable. We don't have any duplicated name nor problems with help.

Reply:
Actually it also checks if we have same metrics in describe and collect, but we don't.